ergotree-ir/src/sigma_protocol/dlog_group.rs

```diff
@@ -168,4 +203,16 @@ mod tests {
             prop_assert_eq![sigma_serialize_roundtrip(&e), e];
         }
     }
+
+    #[test]
+    fn scalar_bigint_roundtrip() {
+        let rand_scalar: Scalar = random_scalar_in_group_range();
+        let as_bigint: BigInt = scalar_to_bigint(rand_scalar);
+        let as_scalar: Option<Scalar> = bigint_to_scalar(as_bigint);
```

Comment:
```suggestion
        assert_eq!(rand_scalar, bigint_to_scalar(as_bigint).unwrap());
```
This will give the same result without matching.